### DISCUSSION TITLE
My map is freeze when I open it in browser.

### DISCUSSION THREAD

**Discussion by brunonunestx** [opengeos/leafmap#730] [General]
Hello! I am getting a problem when I create a simple map using the leafmap module.

I use just this code to create the map:
![image](https://github.com/opengeos/leafmap/assets/90159649/d9ad7e25-b3d5-4d68-be4c-823544f77559)

When I open the file created in the browser (I am using the Google Chrome), I click to move the map and the map freeze. I saw the console and there was the error: `Uncaught Error: Invalid LatLng object: (-13.188724421888733, NaN)`. I try find something about it, but nothing works.

Can someone help me?

**Comment by giswqs** [comment]
Try the following and see if you encounter the same issue. 

```python
import ipyleaflet
m = ipylealet.Map()
m.save('map.html')
```

**Comment by brunonunestx** [comment]
It's working for me, the map is not freezing more. Thank you!
